app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,15 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.HomeFragment.Companion.newInstance
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class HomeActivity : WolmoActivity() {
+    override fun layout(): Int {
```

Comment:
Since is just one line, and considering you are returning something, you can user `=` directly
`override fun layout() = R.layout.activity_base`